src/Provider/Provider.Web/Views/Part2/Qualifications/Qualifications.cshtml

```diff
@@ -0,0 +1,111 @@
+﻿@using Esfa.Recruit.Shared.Web.Mappers
+@using Esfa.Recruit.Vacancies.Client.Domain.Entities
+@model Esfa.Recruit.Provider.Web.ViewModels.Part2.Qualifications.QualificationsViewModel
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">@Model.Title</h1>
+        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState, OrderedFieldNames = Model.OrderedFieldNames }' />
+        
+        <partial name="@PartialNames.ReviewSummary" for="Review" />
+
+        <form asp-route="@RouteNames.Qualifications_Post">
+            <span esfa-validation-message-for="Qualifications" class="govuk-error-message"></span>
+
+            <div esfa-validation-marker-for="QualificationType" class="govuk-form-group">
+                <label asp-for="QualificationType" class="govuk-label govuk-!-font-weight-bold" id="@Anchors.QualificationsHeading">Add qualifications applicants should have</label>
+                <span esfa-validation-message-for="QualificationType" class="govuk-error-message"></span>
+                <select asp-for="QualificationType" class="govuk-select govuk-!-width-one-half">
+                    <option value="">Select from list</option>
+
+                    @foreach (var qualificationType in Model.QualificationTypes)
+                    {
+                        <option value="@qualificationType" selected="@(Model.QualificationType == @qualificationType)">@qualificationType</option>
+                    }
+                </select>
+            </div>
+
+            <div esfa-validation-marker-for="Subject" class="govuk-form-group">
+                <label asp-for="Subject" class="govuk-label govuk-!-font-weight-bold">Subject</label>
+                <span esfa-validation-message-for="Subject" class="govuk-error-message"></span>
+                <input asp-for="Subject" value="@Model.Subject" type="text" class="govuk-input govuk-!-width-one-half" maxlength="50">
+            </div>
+
+            <div esfa-validation-marker-for="Grade" class="govuk-form-group" data-bind="parentvalElement:grade">
```

Comment:
Do we need this data-bind attribute, sounds a bit KO JS'y

Reply:
I'll find out from Steve